UnityProject/Assets/Graphy - Ultimate Stats Monitor/Runtime/GraphyDebugger.cs

```diff
@@ -0,0 +1,570 @@
+        /// </summary>
+        /// <param name="debugVariable"></param>
+        /// <returns></returns>
+        private float GetRequestedValueFromDebugVariable(DebugVariable debugVariable)
+        {
+            switch (debugVariable)
+            {
+                case DebugVariable.Fps:
+                    return m_fpsMonitor != null     ? m_fpsMonitor.CurrentFPS   : 0;
+                case DebugVariable.Fps_Min:
+                    return m_fpsMonitor != null     ? m_fpsMonitor.OnePercentFPS       : 0;
+                case DebugVariable.Fps_Max:
+                    return m_fpsMonitor != null     ? m_fpsMonitor.Zero1PercentFps       : 0;
+                case DebugVariable.Fps_Avg:
+                    return m_fpsMonitor != null     ? m_fpsMonitor.AverageFPS   : 0;
+
+                case DebugVariable.Ram_Allocated:
+                    return m_ramMonitor != null     ? m_ramMonitor.AllocatedRam : 0;
+                case DebugVariable.Ram_Reserved:
+                    return m_ramMonitor != null     ? m_ramMonitor.AllocatedRam : 0;
+                case DebugVariable.Ram_Mono:
+                    return m_ramMonitor != null     ? m_ramMonitor.AllocatedRam : 0;
+
+                case DebugVariable.Audio_DB:
+                    return m_audioMonitor != null   ? m_audioMonitor.MaxDB      : 0;
+
+                default:
+                    return 0;
+
+            }
+        }
+
+        /// <summary>
+        /// Executes the operations in the DebugPacket specified.
+        /// </summary>
+        /// <param name="debugPacket"></param>
+        private void ExecuteOperationsInDebugPacket(DebugPacket debugPacket)
+        {
+            if (debugPacket != null)
+            {
+                if (debugPacket.DebugBreak)
+                {
+                    Debug.Break();
+                }
+
+                if (debugPacket.Message != "")
+                {
+                    string message = "[Graphy] (" + System.DateTime.Now + "): " + debugPacket.Message;
+
+                    switch (debugPacket.MessageType)
+                    {
+                        case MessageType.Log:
+                            Debug.Log(message);
+                            break;
+                        case MessageType.Warning:
+                            Debug.LogWarning(message);
+                            break;
+                        case MessageType.Error:
+                            Debug.LogError(message);
+                            break;
+                    }
+                }
+
+                if (debugPacket.TakeScreenshot)
+                {
+                    string path = debugPacket.ScreenshotFileName + "_" + System.DateTime.Now + ".png";
+                    path = path.Replace("/", "-").Replace(" ", "_").Replace(":", "-");
+
+#if UNITY_2017_1_OR_NEWER
+                    ScreenCapture.CaptureScreenshot(path);
+#else
+                    Application.CaptureScreenshot(path);
```

Comment:
it is part of the assets assembly or is it a separate assembly?

Reply:
Its in its own assembly

Reply:
![image](https://github.com/unitystation/unitystation/assets/34368774/63e1e305-0c5a-4514-aa67-3dbb2e185db1)

github has file filters btw so you can just check for yourself instead of waiting for people to respond over trivial info like this